www/js/app.js

```diff
@@ -640,15 +640,63 @@ function focusPrefixOnHomeKey (event) {
         }, 0);
     }
 }
+
 /**
  * Verifies the given archive and switches contentInjectionMode accourdingly
- * @param {archive} the archive that needs verification
+ * @param {archive} archive the archive that needs verification
```

Comment:
I would change `{archive}` to `{Object}` (with capital letter. Then, instead of `the archive ...` I would change to `The archive ...` (i.e. capitalized `The`).